assets/javascripts/kitten/components/cards/team-card/title.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react'
+import Radium from 'radium'
+import PropTypes from 'prop-types'
+import { Text } from 'kitten/components/typography/text'
+import { Marger } from 'kitten/components/layout/marger'
+
+export class TeamCardTitle extends Component {
+  static propTypes = {
+    firstName: PropTypes.string.isRequired,
```

Comment:
J'aurais été plus générique ici avec 2 props : `title` and `subtitle` qui correspond au 2 zones de textes.

Reply:
On peut même être encore plus libre en laissant libre via le children.